go.mod

```diff
@@ -18,6 +18,9 @@ replace github.com/prometheus/prometheus => github.com/prometheus/prometheus v0.
 // Update when github.com/deepmap/oapi-codegen/v2 is updated to support later versions.
 replace github.com/getkin/kin-openapi => github.com/getkin/kin-openapi v0.122.0
 
+// TODO: remove after merge of https://github.com/grafana/authlib/pull/23
+replace github.com/grafana/authlib => /Users/charandas/go/src/github.com/grafana/authlib
```

Comment:
Had to add `Namespace` to `AccessTokenClaims` or else would have used the branch.